site/components/pages/Pledge/lib/firebase.ts

```diff
@@ -0,0 +1,92 @@
+import * as admin from "firebase-admin";
+import { v4 as uuidv4 } from "uuid";
+
+import { putPledgeParams } from "queries/pledge";
+import { Footprint, Pledge } from "../types";
+import { verifySignature } from ".";
+
+import serviceAccount from "./firebaseServiceAccountFile.json";
```

Comment:
What are your thoughts on establishing a pattern for secrets?
Is this blog similar to how we should be setting the env var?

https://dev.to/vvo/how-to-add-firebase-service-account-json-files-to-vercel-ph5